### PR TITLE
Update generated files

### PR DESCRIPTION
**Pull request by jpeeler**
This was just forgotten in 8887561914b9e4f09ab2110bb6686ce69242910c.

### CODE REVIEW

**Comment by pmorie** [comment]
@jpeeler are there any places in the makefile that need to be updated?

**Comment by jpeeler** [comment]
@pmorie Not that I know of.

**Comment by MHBauer** [comment]
Shouldn't this be failing some builds?

**Comment by duglin** [comment]
I'm kind of wondering why everything built/ran ok w/o this change.  @jpeeler any ideas?

**Comment by MHBauer** [comment]
Sigh, I see. We don't have `--verify-only` for the non-client stuff. We should probably pull the all the generators out of the makefile into their own script so we can reuse them without duplication in the makefile.

**Comment by jpeeler** [comment]
If the generated files simply have stale code, it's not technically wrong in any way. But it sounds like adding `--verify-only` can help.

File: pkg/apis/servicecatalog/v1alpha1/zz_generated.deepcopy.go [hunk 1]
```diff
@@ -191,11 +168,7 @@ func DeepCopy_v1alpha1_BindingStatus(in interface{}, out interface{}, c *convers
 		if in.Conditions != nil {
 			in, out := &in.Conditions, &out.Conditions
 			*out = make([]BindingCondition, len(*in))
-			for i := range *in {
-				if err := DeepCopy_v1alpha1_BindingCondition(&(*in)[i], &(*out)[i], c); err != nil {
-					return err
-				}
-			}
+			copy(*out, *in)
```
Comment (MHBauer) [inline pkg/apis/servicecatalog/v1alpha1/zz_generated.deepcopy.go]: I still get the red version when running the generator. Thoughts?

Comment (pmorie) [inline pkg/apis/servicecatalog/v1alpha1/zz_generated.deepcopy.go]: @MHBauer that is odd.

Comment (pmorie) [inline pkg/apis/servicecatalog/v1alpha1/zz_generated.deepcopy.go]: I'm seeing if I can recreate now.

**Comment by pmorie** [comment]
I get the following under go 1.8.3 when I run `make apiserver-image` on master.

```
$ gs
On branch master
Your branch is ahead of 'origin/master' by 5 commits.
  (use "git push" to publish your local commits)
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

        modified:   pkg/apis/servicecatalog/v1alpha1/types.generated.go
        modified:   pkg/apis/servicecatalog/v1alpha1/zz_generated.conversion.go
        modified:   pkg/apis/servicecatalog/v1alpha1/zz_generated.deepcopy.go
        modified:   pkg/apis/servicecatalog/zz_generated.deepcopy.go
        modified:   pkg/openapi/openapi_generated.go

no changes added to commit (use "git add" and/or "git commit -a")

★ pmorie:~/code/service-catalog (master *) 
$ gd --stat
 pkg/apis/servicecatalog/v1alpha1/types.generated.go         | 379 +++++++++++++++++------------------------------------------------------------------------------------------------------------------------------
 pkg/apis/servicecatalog/v1alpha1/zz_generated.conversion.go |  26 ----------
 pkg/apis/servicecatalog/v1alpha1/zz_generated.deepcopy.go   |  41 ++++------------
 pkg/apis/servicecatalog/zz_generated.deepcopy.go            |  41 ++++------------
 pkg/openapi/openapi_generated.go                            |  33 +------------
 5 files changed, 64 insertions(+), 456 deletions(-)
```

File: pkg/apis/servicecatalog/v1alpha1/zz_generated.deepcopy.go (hunk 1 above)
Comment (MHBauer) [inline pkg/apis/servicecatalog/v1alpha1/zz_generated.deepcopy.go]: my diff patch
https://gist.github.com/MHBauer/155b604c1e881742535b951949815aad

**Comment by pmorie** [comment]
@MHBauer what happens when you do a hard reset to master and run `make apiserver-image`?

**Comment by MHBauer** [comment]
from master
```
% git diff --stat
 pkg/apis/servicecatalog/v1alpha1/types.generated.go         | 379 ++++++++++++++++++++-----------------------------------------------------------------------------------------------------------------------------------------------------
 pkg/apis/servicecatalog/v1alpha1/zz_generated.conversion.go |  26 ------------
 pkg/apis/servicecatalog/v1alpha1/zz_generated.deepcopy.go   |  41 ++++---------------
 pkg/apis/servicecatalog/zz_generated.deepcopy.go            |  41 ++++---------------
 pkg/openapi/openapi_generated.go                            |  33 +--------------
 5 files changed, 64 insertions(+), 456 deletions(-)
```
from pr1115 branch
```
I'm guessing this pr needs to be rebased and regenerated?

**Comment by MHBauer** [comment]
I want to get this fixed and merged because it irritates me and then a new pr for fixing the verify stuff.

**Comment by pmorie** [comment]
I'm not sure I understand what is happening here.  @jpeeler what go version are you using?

**Comment by MHBauer** [comment]
could be a different go version or an old go image. @jpeeler do a `docker pull golang:1.8` ahead of the build. 

We're sort of unintentionally upgraded from 1.8 -> 1.8.3

**Comment by pmorie** [comment]
@MHBauer @jpeeler full disclosure:

I do all my builds with my local go

**Comment by MHBauer** [comment]
version of 1.8 doesn't seem to matter. I've gotten the same results for 1.8.0->1.8.3.

**Comment by pmorie** [comment]
Here's what I get with the current state of this branch:

```
★ pmorie:~/code/service-catalog (pr/1115 *) 
$ gd --stat
 pkg/apis/servicecatalog/v1alpha1/zz_generated.deepcopy.go | 18 +++++++++++++++---
 pkg/apis/servicecatalog/zz_generated.deepcopy.go          | 18 +++++++++++++++---
 pkg/openapi/openapi_generated.go                          | 10 +++++++++-
 3 files changed, 39 insertions(+), 7 deletions(-)

★ pmorie:~/code/service-catalog (pr/1115 *) 
$ gs
On branch pr/1115
Your branch is up-to-date with 'upstream/pr/1115'.
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

        modified:   pkg/apis/servicecatalog/v1alpha1/zz_generated.deepcopy.go
        modified:   pkg/apis/servicecatalog/zz_generated.deepcopy.go
        modified:   pkg/openapi/openapi_generated.go

no changes added to commit (use "git add" and/or "git commit -a")
```

Let's try to get to the bottom of that.  @jpeeler what version of go are you on?

**Comment by jpeeler** [comment]
Apologies, yes I accidentally originally used go 1.7.6 to generate the new files. Would that have broken anything? Either way, I've updated with 1.8.3 now and the diff stat matches both of your results.

**Comment by pmorie** [comment]
Okay, cool.  LGTM

**Pull request merged**